packages/create-wmr/tpl/tsconfig.json

```diff
@@ -15,6 +15,7 @@
 		"allowSyntheticDefaultImports": true,
 		"downlevelIteration": true
 	},
+	"files": ["./node_modules/wmr/index.d.ts"],
```

Comment:
Let's use TypeScripts [`include`](https://www.typescriptlang.org/tsconfig#include) property instead. That we we can add usercode by default too.

Reply:
Ah, apologies, forgot that was the implication with `files`. Fixed!